content/rest/_request.textile

```diff
@@ -0,0 +1,21 @@
+h6(#request).
+  default: request
+
+bq(definition).
+  default: request(String method, String path, Object params, Object body, Object headers, callback("ErrorInfo":/rest/types#error-info err, "HttpPaginatedResponse":/rest/types#http-paginated-response results)))
+
+Makes a REST request to a provided path. This is provided as a convenience for developers who wish to use bleeding edge REST API functionality that is either not documented or is not yet included in the public API, without having to handle authentication, paging, fallback hosts, MsgPack and JSON support, etc. themselves.
+
+h4. Parameters
+
+- method := either @get@ or @post@.<br>__Type: String__
+- path := the path to query.<br>__Type: String__
+- params := (optional) any querystring parameters needed.<br>__Type: Object__
+- body := (optional; for @post@ method only) the body of the request, as anything that can be serialized into JSON, such as an @Object@ or @Array@.<br>__Type: Serializable__
+- headers := (optional) any headers needed. If provided, these will be mixed in with the default library headers.<br>__Type: Object__
+
+h4. Callback result
+
+On success, @results@ contains an "@HttpPaginatedResponse@":/rest/types#http-paginated-response containing the @statusCode@ and a @success@ boolean, @headers@, and an @items@ array containing the current page of results. It supports pagination using "@next@":#paginated-result and "@first@":#paginated-result methods, identically to "@PaginatedResult@":/rest/types#paginated-result.
```

Comment:
Are we being clear enough here what constitutes success? Reading this I might think a completed request that returns an "error" statusCode such as 400 would be a success.

Are we going to define what constitutes a success error code for every method? or any 2xx ?